Example/Controllers/ContactsListSettingsTableViewController.swift

```diff
@@ -59,11 +109,12 @@ class ContactsListSettingsTableViewController: UITableViewController {
 
     func addCustomContactId(title: String, message: String) {
         DispatchQueue.main.async {
-            self.getInputFromAlertWithTextField(title: title, message: message) { (_, textField) in
+            self.getInputFromAlertWithTextField(title: title, message: message, textFieldDefaultValue: UUID().uuidString) { (_, textField) in
```

Comment:
As we usually create contacts with UUIDs, I replaced the empty textfields we have in contact creation by a UUID prefilled one